autode/hessians.py

```diff
@@ -400,13 +389,42 @@ def frequencies_proj(self) -> List[Frequency]:
                 "Could not calculate projected frequencies, must "
                 "have atoms set"
             )
-
         n_tr = self.n_tr  # Number of translational+rotational modes
-        lambdas = np.linalg.eigvalsh(self._proj_mass_weighted[n_tr:, n_tr:])
 
-        trans_rot_freqs = [Frequency(0.0) for _ in range(n_tr)]
+        H = self._proj_mass_weighted
+        norms = np.linalg.norm(H, axis=0)
```

Comment:
I am curious if this will work, because the projection forces the eigenvalue to be zero, but the norm may or not be zero? Or does it have to be? I am not so familiar with linear algebra to be able to tell 😅

Reply:
@t-young31 I tried the code with an ethane molecule and XTB hessian - but the projected matrix does not seem to be block zero at all. Here is the projected mass weighted matrix: [projected.txt](https://github.com/user-attachments/files/18181037/projected.txt) and here is the xyz file: [test.xyz.txt](https://github.com/user-attachments/files/18181055/test.xyz.txt). I am not sure if the projection is actually working. I will try to go through the original literature papers to try to make sense of this -- it might be that the stackoverflow page had some errors 😕

Reply:
for water it looks okay

```
-0.00  0.00 -0.00  0.00  0.00  0.00 -0.00  0.00 -0.01
 0.00 -0.00  0.00 -0.00  0.00 -0.00 -0.00  0.00 -0.00
-0.00  0.00  0.00 -0.02  0.00  0.00  0.00 -0.00  0.00
 0.00 -0.00 -0.02  0.10 -0.00 -0.00 -0.00  0.00 -0.00
 0.00  0.00  0.00 -0.00 -0.10 -0.00  0.00 -0.00 -0.00
 0.00 -0.00  0.00 -0.00 -0.00 -0.00  0.00 -0.00  0.00
-0.00 -0.00  0.00 -0.00  0.00  0.00  3.92 -1.55  0.07
 0.00  0.00 -0.00  0.00 -0.00 -0.00 -1.55  1.63  0.15
-0.01 -0.00  0.00 -0.00 -0.00  0.00  0.07  0.15  4.72
```

but as you say, for ethane it looks quite a bit worse 😕  (using `np.savetxt("tmp.txt", H/1e9, fmt='%5.2f')`)